s/frontend/components/banned-users/index.css.ts

```diff
@@ -5,6 +5,12 @@ export default css`
 	width: 100%;
 	padding: 20px;
 }
+img{
+	width:100px;
```

Comment:
please mind the whitespace, consistency with the rest of the codebase is preferable